reactive/flannel.py

```diff
@@ -176,6 +176,43 @@ def reconfigure_docker_for_sdn():
     set_state('flannel.bridge.configured')
 
 
+@when_any('config.http_proxy.changed', 'config.https_proxy.changed')
+def rerender_service_template():
+    ''' If we change proxy settings, re-render the bootstrap service definition
+    and attempt to resume where we left off.  '''
+
+    # Note: At this point if we hijack the workload daemon, heavy fisted
+    # reprocussions will occur, like disruption  of services.
+
+    codename = host.lsb_release()['DISTRIB_CODENAME']
+    # by default, dont reboot the daemon unless we have previously rendered
+    # system files.
+    reboot = False
+
+    # Deterministic method to probe if we actually need to restart the
+    # daemon.
+    if(os.path.exists('/lib/systemd/system/bootstrap-docker.service') or
```

Comment:
You can remove the reboot = False as well, and just evaluate this one time like so:

```python
reboot = (os.path.exists('/lib/systemd/system/bootstrap-docker.service') or
          os.path.exists('/etc/init/bootstrap-docker.conf'))
```